client-src/index.js

```diff
@@ -133,8 +133,18 @@ const onSocketMessage = {
 
     reloadApp(options, status);
   },
-  'content-changed': function contentChanged() {
-    log.info('Content base changed. Reloading...');
+  // TODO: remove in v5 in favor of 'static-changed'
+  'content-changed': function contentChanged(file) {
+    log.info(
+      `${file || 'Content'} from static directory was changed. Reloading...`
+    );
+
+    self.location.reload();
+  },
+  'static-changed': function staticChanged(file) {
+    log.info(
+      `${file || 'Content'} from static directory was changed. Reloading...`
```

Comment:
Can we use `${file ? `"${file}"` : 'Content'} from static directory was changed. Reloading...`? Better to use `"` for special words/paths/names/etc, it's much better for reading

Reply:
Updated 👍🏻